main.go

```diff
@@ -37,12 +36,13 @@ func init() {
 
 		ex, err := os.Executable()
 		if err != nil {
-			panic(err)
+			fmt.Println(err)
+			return err
 		}
 		exPath := filepath.Dir(ex)
 
 		var savedComs savedCommands
-		data, err := ioutil.ReadFile(filepath.Join(exPath,"saved_commands.json"))
+		data, err := ioutil.ReadFile(filepath.Join(exPath, "saved_commands.json"))
```

Comment:
FWIW, I believe `ioutil.ReadFile()` is deprecated in favor of `os.ReadFile()`. I noticed that in a lint on something else and remembered seeing it here, so figured I'd mention it.

Reply:
Fixed!